packages/inter-protocol/test/vaultFactory/vault-collateralization.test.js

```diff
@@ -15,6 +16,46 @@ test.before(async t => {
   trace(t, 'CONTEXT');
 });
 
+test('totalDebt does not include compoundedInterest', async t => {
```

Comment:
"not" describes the bug but "does include" is the expected behavior this is confirming. or perhaps more explicitly,
```suggestion
test('totalDebt calculation includes compoundedInterest', async t => {
```